OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -291,11 +292,79 @@ public override bool HandleKeyPress(KeyInput e)
 					return ToggleMute();
 				else if (key == Game.Settings.Keys.TogglePlayerStanceColorsKey)
 					return TogglePlayerStanceColors();
+				else if (key == Game.Settings.Keys.NextTrack)
```

Comment:
Implementing these in `WorldInteractionControllerWidget` means that they won't work in any of the menus.  It would be better if these hotkeys could work anywhere where music is playing.

You'll also need to be careful to not break the special case background music on the shellmap, on game completion, and ambient effects (rain etc).